City-of-Helsinki/hel-fi-drupal-grants
# Fix form required

# [AU-XX](https://helsinkisolutionoffice.atlassian.net/browse/UHF-0000)(where was this ticket?)


Osoite fields are required with communities but not with private persons. This led to issues where Osoite field was not required when needed or required when it was not. 

This fixes that. As long as there's no address required for private persons. Ever.

## What was done


* Added conditional required for private persons in code.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout fix-form-required`
  * `make fresh`
* Run `make drush-cr`

## How to test


*  Open projektiavustushakemus 
* With private person address should not be required, otherwise it should.
* Check that code follows our standards
